contrib/ansible/roles/common/tasks/system-proxy.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Proxy | Install system http proxy environment variable
+  lineinfile: dest=/etc/environment regexp=^HTTP_PROXY= line=HTTP_PROXY="{{ system_http_proxy }}"
```

Comment:
wow ok. so now I remember what is effected. So Kubectl is effected connecting from a host with a proxy set in `/etc/environment`.. I logged a bug on it a while back. #6808. So if you are on a node with a proxy set and try to connect to the apiserver it goes through the proxy. The master node is unaffected since the default `kubectl`  connection server is `127.0.0.1` that is in the `no_proxy` environment variable.

As far as the server components I have had no connection issues since they do not grab the `/etc/environment` as systemd services. 

I will edit this PR for only adding the new CentOS repo.